application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -57,6 +64,7 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
         String userID = (Objects.requireNonNull(event.getMember()).getId());
         TeXFormula formula;
         try {
+            latex = convertInlineLatexToFull(latex);
```

Comment:
What if the message doesnt even contain any inline latex? The input should be considered a full latex already then, i.e. ur method should not be executed in that case. A comment might also help to explain whats going on.

I.e.
```java
if (latex.contains("$")) {
  latex = convertInlineLatexToFull(latex);
}
```

Reply:
Yes but it also shouldnt. Like, the `/tex` command should still be able to be used as it was originally intended. So like
`/tex \frac{x}{2}`, only with latex and no actual text.

So the command has to support both:
* just latex, but without `$...$`
* mixed text with inline latex, i.e. with `$...$`